keras_cv/models/backbones/efficientnetv2/efficientnet_v2_backbone.py

```diff
@@ -592,42 +572,15 @@ class EfficientNetV2(keras.Model):
         activation: activation function.
         blocks_args: list of dicts, parameters to construct block modules.
         model_name: string, model name.
-        weights: one of `None` (random initialization),
-            or the path to the weights file to be loaded.
         input_shape: optional shape tuple, defaults to (None, None, 3).
         input_tensor: optional Keras tensor (i.e. output of `layers.Input()`)
             to use as image input for the model.
-        pooling: optional pooling mode for feature extraction
-            when `include_top` is `False`.
-            - `None` means that the output of the model will be
-                the 4D tensor output of the
-                last convolutional layer.
-            - `avg` means that global average pooling
-                will be applied to the output of the
-                last convolutional layer, and thus
-                the output of the model will be a 2D tensor.
-            - `max` means that global max pooling will
-                be applied.
-        num_classes: optional number of classes to classify images
-            into, only to be specified if `include_top` is True.
-        classifier_activation: A `str` or callable. The activation function to
-            use on the "top" layer. Ignored unless `include_top=True`. Set
-            `classifier_activation=None` to return the logits of the "top"
-            layer.
-
-    Returns:
-      A `keras.Model` instance.
-    Raises:
-      ValueError: in case of invalid argument for `weights`,
-        or invalid input shape.
-      ValueError: if `classifier_activation` is not `"softmax"` or `None` when
-        using a pretrained top layer.
     """
 
     def __init__(
         self,
+        *,
         include_rescaling,
-        include_top,
         width_coefficient,
```

Comment:
@jbischof what do you want to do about things like "width_coefficient" and "depth_coefficient" that are generally only used by the wrapping classes?  We typically dont expect users to configure them; but I guess we just document them and leave them in?